tests/api/v1/endpoints/test_dataset_endpoints.py

```diff
@@ -34,7 +34,7 @@ def _reject_key(dict: Dict, key: str) -> Dict:
 def test_example_datasets(example_datasets):
     """Ensure the test fixture loads the right sample data"""
     assert example_datasets
-    assert len(example_datasets) == 3
+    assert len(example_datasets) == 4
```

Comment:
This is maybe part of the wider testing changes. I'm not sure what value adding every new dataset to `example_datasets` brings us vs. loading them individually. Do you have a reason here or is it to respect convention?

Reply:
this existing test just loads in all datasets, so I just changed it because tests were failing - I can just get rid of this test.